src/Symfony/Component/Console/Tests/Helper/QuestionHelperTest.php

```diff
@@ -914,6 +914,10 @@ public function testTraversableMultiselectAutocomplete()
 
     public function testAutocompleteMoveCursorBackwards()
     {
+        if (!Terminal::hasSttyAvailable()) {
```

Comment:
we already have this check on the other autocomplete related tests, seems like it was forgotten here